railties/lib/rails/application/configuration.rb

```diff
@@ -245,8 +245,8 @@ def load_defaults(target_version)
           end
 
           if respond_to?(:action_controller)
+            action_controller.raise_on_missing_callback_actions = Rails.env.development? || Rails.env.test?
```

Comment:
Seems like we could rely on the `config/environments/*.rb` for this and set it to false by default?

Reply:
Good point, the setting was already added to the development/test config templates, so setting this to false.